config/initializers/ios_apn.rb

```diff
@@ -0,0 +1,9 @@
+if Rails.env.production?
+  IOS_APN = ENV['IOS_APN'] || Houston::Client.production
+  IOS_APN.certificate = File.read('config/ios_production_certificate.txt')
+  IOS_APN.passphrase = File.read('config/ios_production_passphrase.txt')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.